packages/ui/VirtualList/VirtualListBasic.js

```diff
@@ -269,8 +269,8 @@ class VirtualListBasic extends Component {
 		direction: 'vertical',
 		overhang: 3,
 		pageScroll: false,
-		spacing: 0,
-		type: 'JS'
+		scrollMode: 'translate',
```

Comment:
The default value would be `native`, right?

Reply:
I didn't notice that the default value for ui/VirtualList, ui/Scroller `scrollMode` is 'translate'. Please ignore my comment.